runtime/protocol/http-client-engines/http-client-engine-ktor/jvm/src/aws/smithy/kotlin/runtime/http/engine/ktor/KtorRequestAdapter.kt

```diff
@@ -50,6 +51,7 @@ internal class KtorRequestAdapter(
         }
     }
 
+    @InternalAPI
     fun toBuilder(): KtorRequestBuilder {
```

Comment:
We shouldn't propagate `@InternalAPI` we should optin at the sites that it is used.

Reply:
Thanks, yeah wrong annotations here.